### PR TITLE
Automatically sanitize Dev databases.

### PR DESCRIPTION
**Pull request by nader77**
#86

### CODE REVIEW

File: app/templates/scripts/db_sanitization.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+// Don't ever santize the database on the live environment. Doing so would
+// destroy the canonical version of the data.
+if (defined('PANTHEON_ENVIRONMENT') && (PANTHEON_ENVIRONMENT !== 'live')) {
+  // Bootstrap Drupal using the same technique as is in index.php.
+  define('DRUPAL_ROOT', $_SERVER['DOCUMENT_ROOT']);
+  require_once DRUPAL_ROOT . '/includes/bootstrap.inc';
+  drupal_bootstrap(DRUPAL_BOOTSTRAP_DATABASE);
+  // From http://crackingdrupal.com/blog/greggles/creating-sanitized-drupal-database-dump#comment-164
+  db_query("UPDATE users SET mail = CONCAT(name, '@test'), init = CONCAT(name, '@test'), pass = MD5(CONCAT('1234', name));");
```
Comment (amitaibu) [inline app/templates/scripts/db_sanitization.php]: I suspect the MD5 part for the password won't really work on D7 site - it looks like D6. I think we can remove the pass part.

**Comment by amitaibu** [comment]
Thanks for working on this!

File: app/templates/scripts/db_sanitization.php (hunk 1 above)
Comment (nader77) [inline app/templates/scripts/db_sanitization.php]: Alright, I will create a demo website on Pantheon to test it out.

**Comment by nader77** [comment]
I was thinking about creating another PR for automatically [revert features](https://github.com/pantheon-systems/quicksilver-examples/tree/master/drush_revert_features), What do you think ?

**Comment by amitaibu** [comment]
Sounds good :)

**Comment by nader77** [comment]
@amitaibu Ready for code review.

1. Applying `pantheon.yml`:
sanitize-db_ _-bash_ _178x40

2. Adding user in test:
nader___sanitize-db

3. User email after clone:
nader___sanitize-db

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request merged**